www/assets/css/theme.css

```diff
@@ -378,3 +378,99 @@ code { /* character, digrams, trigrams */
     color: var(--fg-banner);
   }
 }
+
+
+/**
+ * ╭───────────────────────────────────────────────────────────────╮
+ * │                          Comments                             │
+ * ╰───────────────────────────────────────────────────────────────╯
+**/
```

Comment:
I think this could be in a separate CSS file, e.g. `comments.css`, as it’s only used by single-page articles.